packaging/artifact-registry-yum-plugin.spec

```diff
@@ -33,6 +33,7 @@ Requires: python3-pyasn1
 Requires: python3-pyasn1-modules
 Requires: python3-six
 
+BuildRequires: python-rpm-macros
```

Comment:
python3-rpm-macros?

Reply:
the way the unversioned macros work is they require you to define the python version you're using, which you can see in this spec file with `__define`. it's still supported, and like i said, python3-rpm-macros will depend on and thus install python-rpm-macros

that said, to move to the specific version we would remove the define and update all macros in the spec to use the python3-specific versions

Reply:
Up to you- I just see it as a future proofing thing.